package.json

```diff
@@ -105,6 +105,7 @@
 		"tempy": "0.2.1",
 		"valid-url": "1.0.9",
 		"wamp-socket-cluster": "2.0.0-beta.4",
+		"yargs": "13.2.2",
```

Comment:
Only yargs should be in this PR - ` feature/improve_overall_configuration` is not up to date.

Reply:
The only addition is this PR is `yargs`, rest were formatted by `npm install`

Reply:
I see only `yargs` included in this PR, so im resolving this